### DISCUSSION TITLE
Notion, Slack

### DISCUSSION THREAD

**Discussion by pulapulkaaaaa** [dailydotdev/daily#1223] [Feature Requests]
I would like to integrate daily.dev or any other platform integration or to make browser extension way more better

**Comment by idoshamun** [comment]
Hi @pulapulkaaaaa,

What would you expect from such integrations?
And what would you like to see when you say "make the browser extension way better"?

**Comment by pulapulkaaaaa** [comment]
I use both platforms a lot in my day in this case I can give you idea to embed posts into notion or slack so you don't lose views metrics and other statistics. Also as per extension in Mozilla (this is what I personally use) I love an integration like Pocket where posts are below of main search bar and shortcuts 
![dfghdrgsfhdfghdfg](https://github.com/dailydotdev/daily/assets/155702650/4bdcef3a-71ef-4d81-8b05-cad491d3f626)

**Reply by pulapulkaaaaa** [reply]
instead of pocket posts can be daily dev
